source/faq.txt

```diff
@@ -183,6 +183,18 @@ The error message consists of multiple parts:
      can use tools like ``openssl s_client`` to debug TLS/SSL-related
      certificate problems.
 
+.. _csharp-faq-linq-vs-builder:
+
+Should I Use LINQ or Builder Classes When Querying for Documents?
+-----------------------------------------------------------------
+
+If you're used to programming in C#, consider using LINQ because of its similar feel
+to programming in native C#. If you have prior experience with other MongoDB drivers,
```

Comment:
```suggestion
If you're used to programming in {+language+}, consider using LINQ because of its similar feel
to programming in native {+language+}. If you have prior experience with other MongoDB drivers,
```